README.md

```diff
@@ -13,6 +13,24 @@ The driver is compatible across the entire line of UR robots -- from 3 kg payloa
 Check also [presentations and videos](ur_robot_driver/doc/resources/README.md) about this driver.
 
 
+## Installation
```

Comment:
I think it makes sense to combine this with the compilation instructions such that you have "installation" and then as a sub chapter "Install from binary packages" and "Build from source" or similar. Also, the binary installation part should get a note, that this will not necessarily reflect the latest state on Gihub, as those binary packages are created at discrete tagged points (linked to the actual release tags page).